RecoPixelVertexing/PixelTriplets/src/CAGraph.h

```diff
@@ -42,6 +48,14 @@ struct CALayerPair {
 };
 
 struct CAGraph {
+  int getLayerId(const std::string &layerName) {
+    for (auto thisLayer : theLayers) {
```

Comment:
```const auto &```

Reply:
No problem! Just to be sure, you mean for `this Layer`, right?

Reply:
Yes, thank you!